src/components/QuickOpenModal.js

```diff
@@ -50,6 +50,12 @@ type Props = {
   closeQuickOpen: () => void
 };
 
+const styles = {
```

Comment:
We should really try to take this into its own css. I know it didn't work out the first time but having css styling in our js files is going to make the code really ugly.